nservicebus/sagas/timeouts.md

```diff
@@ -34,29 +34,28 @@ Multiple timeouts can be requested when processing a message. The individual tim
 
 snippet: saga-multiple-timeouts
 
+NOTE: Sending a timeout with the same data multiple times will result in that timeout being processed multiple times by the saga.
+
 ## Revoking timeouts
 
 A timeout that has been scheduled cannot be revoked. This means that when the timeout timestamp has elapsed then this timeout message will be queued and then processed.
 
 A timeout is a regular message and once requested, a timeout message can already be in transit or queued. Once that has happened there is no way to revoke (delete) a timeout. It is common to perform a state check in a timeout handler to see if the timeout is still applicable for processing.
 
-NOTE: Sending a timeout with the same data multiple times will result in that timeout being processed multiple times by the saga.
-
-
 ## Completed Sagas
 
-It is possible for a timeout to be queued after its saga has completed. Because a timeout is tied to a specific saga instance it will be ignored once the saga instance is completed.
+It is possible for a timeout to be queued after its saga has been completed. Because a timeout is tied to a specific saga instance it will be ignored once the saga instance is completed.
 
-NOTE: If a saga is created with a previously used saga identifier, the timeout mechanism will not treat it as the previous saga. As a result any residual timeout messages for the previous, now completed, saga will not be processed by this new saga instance that shares the same identifier. The existing timeout message will be ignored.
+NOTE: If a saga is created with a previously used saga identifier, the timeout mechanism will not treat it as the previous saga. As a result, any residual timeout messages for the previous, now completed, saga will not be processed by this new saga instance that shares the same identifier. The existing timeout message will be ignored.
 
 
 ## Timeout state
 
-The state parameter provides a way to pass state to the Sagas timeout handle method. This is useful when many timeouts of the same "type" that will be active at the same time. One example of this would be to pass in some ID that uniquely identifies the timeout eg: `.RequestTimeout(new OrderNoLongerEligibleForBonus{OrderId = "xyz"})`. With this state passed to the timeout handler it can now decrement the bonus correctly by looking up the order value from saga state using the provided id.
+The state parameter provides a way to pass the state to the Sagas timeout handle method. This is useful when many timeouts of the same "type" will be active at the same time. One example of this would be to pass in some ID that uniquely identifies the timeout eg: `.RequestTimeout(new OrderNoLongerEligibleForBonus{OrderId = "xyz"})`. With this state passed to the timeout handler it can now decrement the bonus correctly by looking up the order value from the saga state using the provided id.
```

Comment:
I would leave the first instance as-is. Using "the" there suggests there is only a single "state" to be passed to a timeout (or worse, it may even suggest that the entire saga state should be passed), but since we may call multiple timeouts, passing a differently shaped "state" to each one, there be many "states" which we could refer to. It's a bit nuanced, but I think just "state" is better here.

There is, of course, only one _saga state_ though, so "the" makes sense there.

Moreover, "state" may be bad choice of name for the thing we pass to a timeout, but that's battle to be fought another day (and don't get me started on "timeout" or `RequestTimeout` 😉).

```suggestion
The state parameter provides a way to pass state to the Sagas timeout handle method. This is useful when many timeouts of the same "type" will be active at the same time. One example of this would be to pass in some ID that uniquely identifies the timeout eg: `.RequestTimeout(new OrderNoLongerEligibleForBonus{OrderId = "xyz"})`. With this state passed to the timeout handler it can now decrement the bonus correctly by looking up the order value from the saga state using the provided id.
```